firestore/FirestoreExample/LocalCollection.swift

```diff
@@ -67,7 +63,7 @@ final class LocalCollection<T: DocumentSerializable> {
         return
       }
       let models = snapshot.documents.map { (document) -> T in
-        if let model = T(dictionary: document.data()) {
+        if let model = try? document.data(as: T.self) {
```

Comment:
Since this error is being handled below, this `if let ... else` should be changed into a `do { }` statement.